<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1207</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LIFE-FIT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="settings_icon.qrc">
    <normaloff>:/new/settings/resources/life_icon.ico</normaloff>:/new/settings/resources/life_icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 242, 246);
border-top-color: rgb(53, 64, 82);
;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="left_menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>260</width>
      <height>167772</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 64, 82);
color: rgb(163, 180, 200);</string>
    </property>
    <widget class="QPushButton" name="settings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>40</width>
       <height>39</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>39</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="settings_icon.qrc">
       <normaloff>:/new/settings/resources/settings.png</normaloff>:/new/settings/resources/settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>39</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="avatar">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>125</y>
       <width>125</width>
       <height>125</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>125</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/settings/resources/avatar-df.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="nome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>241</width>
       <height>35</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>241</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RICCARDO GIORATO</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="eta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ETA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OBBIETTIVO  questa settimana:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logout">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>15</y>
       <width>35</width>
       <height>39</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>39</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="settings_icon.qrc">
       <normaloff>:/new/settings/resources/quit.png</normaloff>:/new/settings/resources/quit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>39</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="top_header" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>16777215</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-top: 3px solid  rgb(53, 64, 82);
border-bottom: 3px solid  rgb(214, 220, 224);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>46</number>
     </property>
     <item>
      <widget class="QLabel" name="app_name">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(25, 173, 250);
border: none;</string>
       </property>
       <property name="text">
        <string>LIFE-FIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="content" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>1021</width>
      <height>1181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QWidget" name="right_progress" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>530</y>
       <width>601</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="CircularProgress" name="progressoMese" native="true">
      <property name="geometry">
       <rect>
        <x>408</x>
        <y>50</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="CircularProgress" name="progressoSonno" native="true">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>50</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="CircularProgress" name="progressoMovim" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="calendario_titolo_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 60, 75);</string>
      </property>
      <property name="text">
       <string>Passi</string>
      </property>
     </widget>
     <widget class="QLabel" name="calendario_titolo_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 60, 75);</string>
      </property>
      <property name="text">
       <string>Sonno</string>
      </property>
     </widget>
     <widget class="QLabel" name="calendario_titolo_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(68, 60, 75);</string>
      </property>
      <property name="text">
       <string>Progressi complessivi mese</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="right_calendar" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>600</width>
       <height>451</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>451</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(67, 72, 88);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::LongDayNames</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="calendario_titolo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 60, 75);</string>
     </property>
     <property name="text">
      <string>Calendario</string>
     </property>
    </widget>
    <widget class="QLabel" name="amici_t">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 60, 75);</string>
     </property>
     <property name="text">
      <string>Classifica Amici</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aggiungi_dati">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>810</y>
       <width>411</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px;
border-style: solid;
border-radius: 30px;
border-color: rgb(53,64,82);
background: rgb(53,64,82);
padding-left: 5px;
color: white;
</string>
     </property>
     <property name="text">
      <string>Aggiungi nuovi dati Attività Movimento e Sonno</string>
     </property>
    </widget>
    <widget class="QLabel" name="no_data_img">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>901</width>
       <height>781</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>70</y>
       <width>261</width>
       <height>731</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="friend_list">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <zorder>no_data_img</zorder>
    <zorder>right_calendar</zorder>
    <zorder>calendario_titolo</zorder>
    <zorder>amici_t</zorder>
    <zorder>right_progress</zorder>
    <zorder>aggiungi_dati</zorder>
    <zorder>verticalLayoutWidget_2</zorder>
   </widget>
   <widget class="QWidget" name="content2" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>941</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <widget class="QWidget" name="right_settings" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>45</y>
       <width>700</width>
       <height>450</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>450</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255); </string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>220</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>cambia password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>cambia data di nascita</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>cambia sesso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="impostazioni">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(68, 60, 75);</string>
     </property>
     <property name="text">
      <string>Impostazioni</string>
     </property>
    </widget>
   </widget>
   <zorder>left_menu</zorder>
   <zorder>top_header</zorder>
   <zorder>content2</zorder>
   <zorder>content</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CircularProgress</class>
   <extends>QWidget</extends>
   <header>circularprogress.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="settings_icon.qrc"/>
 </resources>
 <connections/>
</ui>
